5.0/en/0x12-V3-Session-management.md

```diff
@@ -17,6 +17,7 @@ As previously noted, these requirements have been adapted to be a compliant subs
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
 | **3.1.1** | [DELETED, MERGED TO 8.3.1] | | | | | |
 | **3.1.2** | [ADDED] Verify that the application performs all session token verification using a trusted, back-end service. | ✓ | ✓ | ✓ | 603 | |
+| **3.1.3** | [MOVED FROM 3.5.2, MODIFIED] Verify that the application uses either **cryptographically signed or opaque tokens** for session management. Static API secrets and keys should be avoided. | ✓ | ✓ | ✓ | 798 | 7.1 |
```

Comment:
> `**cryptographically signed or opaque tokens**`

I don't think it is a good idea to use markdown in the table. It messes with CSV export, for example. Also, I don't think the emphasis adds anything here.

Reply:
Great point @Sjord! I have removed that